web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
+    .service([
+        '$http',
+        ProjectConfig.componentName,
+        API
```

Comment:
I expose it for easier mocking, I think it's a bit easier to just pass in a mock module than to use the `injector()` syntax.

I'm not bothered too much with this syntax:
1) you have one indendation level less if you define it as a function
2) you still see the modules being imported where the function is defined

I would like to keep it this way